astronomer/providers/dbt/cloud/sensors/dbt.py

```diff
@@ -42,7 +51,7 @@ def execute(self, context: "Context") -> None:
                 run_id=self.run_id,
                 conn_id=self.dbt_cloud_conn_id,
                 account_id=self.account_id,
-                poll_interval=self.poll_interval,
+                poll_interval=self.poke_interval,
```

Comment:
```suggestion
                poke_interval=self.poke_interval,
```

Reply:
Ignore EMR and dbt sensor renaming of `poll_interval` in this PR, because sensor triggers inherits from the operator triggers class, so it will covered in the next PR which can cover the renaming of `poll_interval` to `poke_interval`